src/main/java/org/opensearch/performanceanalyzer/decisionmaker/deciders/CacheHealthDecider.java

```diff
@@ -77,8 +77,10 @@ public Decision operate() {
         counter = 0;
 
         for (final ResourceEnum cacheType : modifyCacheActionPriorityList) {
-            getActionsFromRca(cacheTypeBaseClusterRcaMap.get(cacheType), impactedNodes)
-                    .forEach(decision::addAction);
+            BaseClusterRca baseClusterRcaMap = cacheTypeBaseClusterRcaMap.get(cacheType);
+            if (baseClusterRcaMap == null)
+                continue;
```

Comment:
return?

Reply:
we would just need to go to the next cache action in that case right ? if we return, then we no longer loop through the cache action priority list

Reply:
You are right, it would be continue.